zerver/webhooks/jira/view.py

```diff
@@ -17,12 +17,25 @@
     "attachment_created",
     "issuelink_created",
     "issuelink_deleted",
+    # issuelink_created and issuelink_deleted events are ignored because creating or removing
+    # JIRA issue link doesn't trigger any events. This means that the operation can't be used
+    # to trigger notification or posts.
     "jira:version_released",
-    "jira:worklog_updated",
+    # jira:version_released event is ignored because it sometimes provide inaccurate
+    # information. Here is the link if you want to check
+    # https://jira.atlassian.com/browse/JRACLOUD-75785?jql=text%20~%20%22%5C%22jira%3Aversion_released%5C%22%22 .
     "sprint_closed",
+    # sprint_closed event is ignored because Scope of Webhook event Sprint Closed doesn't get
+    # limited to the JQL specified. If you want to go deep you can check this link
+    # https://jira.atlassian.com/browse/JRASERVER-69135?jql=text%20~%20%22%5C%22sprint_closed%5C%22%22 .
     "sprint_started",
+    # sprint_started event is ignored because when a Sprint is started with a future time
+    # and/or date in the 'Start Date', it's shown in Agile as active, but changes made to its
+    # scope aren't recorded until the selected date/time passes.
     "worklog_created",
     "worklog_updated",
+    # workload_created and workload_update events is ignored because JQL filter for Webhooks
+    # doesn't work correctly when Comment or Worklog related events are fired.
 ]
```

Comment:
I'm not sure these are the reasons why we currently ignore these events -- I think of this list as events we haven't implemented, and the types of explanations that make sense to me are things like "adding links between issues is unlikely to be an interesting notification".  Whereas for something like `sprint_closed`, I'd expect the note to be that we should probably support that event, but it's rare and we haven't implemented it yet.

Reply:
Thanks for the feedback.